sdks/go/pkg/beam/testing/passert/equals.go

```diff
@@ -40,6 +40,18 @@ func Equals(s beam.Scope, col beam.PCollection, values ...interface{}) beam.PCol
 	return equals(subScope, col, other)
 }
 
+// EqualsList verifies that the given collection has the same values as a
+// given list, under coder equality. The values must be provided as an
+// array or slice.
+func EqualsList(s beam.Scope, col beam.PCollection, list interface{}) beam.PCollection {
```

Comment:
Consider adding an additional statement saying that this is equivalent to using a beam.CreateList PCollection as an expected value to Equals.
This does help save a line or two for additional semantic clarity, but noting the equivalence helps make beam more accessible.